### DISCUSSION TITLE
i have finetuned ppocrV5 recognisition model but it is not recognising

### DISCUSSION THREAD

**Discussion by Dpat123** [PaddlePaddle/PaddleOCR#17649] [Q&A]
i have done 75 epoch learning and learning rate was 0.0001 very low 
and i have dataset of 300 images .
could you please let me know how many images required for fine tuning .
